### ISSUE TITLE
PM2.5 forecasting model

### ISSUE THREAD

**Issue by Muhongfan** [Muhongfan/printguard-ai#11]
# PM2.5 forecasting model

## Description

Develop a model that predicts short-term PM2.5 levels.

Prediction horizon: 5 minutes.

## Model Options

LightGBM
ARIMA baseline

## Tasks

- Prepare training dataset
- Train prediction model
- Evaluate prediction accuracy

## Acceptance Criteria

- Model predicts PM2.5 levels
- Evaluation metrics documented
